Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.csproj

```diff
@@ -4,14 +4,46 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <DefaultItemExcludes>$(DefaultItemExcludes);.*;node_modules\**</DefaultItemExcludes>
     <IsPublishable>false</IsPublishable>
-    <InitNodeModulesStampFile>node_modules\.install-stamp</InitNodeModulesStampFile>
+    <!-- The NuGet package reference to Lombiq.Npm.Targets would run those targets during the build of this project,
```

Comment:
Isn't the source build running them too?

Reply:
Why are we not including them when doing the source build though? In effect, NE should work the same from source and NuGet too. If we need NPM Targets for that, then I don't understand why we'd rely on the NPM Targets' targets file to be active during NuGet build but not during source build.

Reply:
NE does not need NPM Targets itself. NE needs NPM Targets' `.props` and `.targets` files to build consumer projects.

Now, the NuGet package reference to NPM Targets automatically includes NPM Targets' `.props` and `.targets` files when NE is built, which we are working around here. I don't know of a better way to do that. `<ExcludeAssets>build</ExcludeAssets>` would probably achieve that, but we need the build assets (`.props` and `.targets` files) for the consumer projects.

Reply:
I see. Then elaborate this in the comments something along the lines of, how I understand, "during the build of this project in the beginning (?) automatically. However, we want to run them at specific stages. Thus, we need to disable the automatic execution here."

Reply:
It's not really about the beginning or other build steps. It's about **this project** being build vs. the **consumer project** being built, and we want the execution of the NPM Targets only in the latter case, **not** the former.

But maybe this will become obsolete once I've figured out the right combination of `PrivateAssets` and `IncludeAssets`.

Reply:
In any case, if this remains, please document the reasons for having it.

Reply:
I've been able to remove this overrides now 🎉

Reply:
I've had to restore the overrides, and I've documented why. It's a separate block at the bottom of the file now.